Formula/p/pivit.rb

```diff
@@ -0,0 +1,26 @@
+class Pivit < Formula
+  desc "Sign and verify data using hardware (Yubikey) backed x509 certificates (PIV)"
+  homepage "https://github.com/cashapp/pivit"
+  url "https://github.com/cashapp/pivit/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "67a2f353c820aad9c9a4cc937ba256c75cb36044baa0578b74a1bfbdf4e6bd90"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  on_linux do
+    depends_on "pcsc-lite"
+  end
+
+  def install
+    system "make", "pivit"
+    bin.install "pivit"
+  end
+
+  test do
+    output = pipe_output("#{bin}/pivit -p 2>&1").strip
```

Comment:
```suggestion
    output = shell_output("#{bin}/pivit -p 2>&1").strip
```

Reply:
Fixed, and added `1` as the expected exit status